core/slim_globals.h

```diff
@@ -354,6 +354,48 @@ extern EidosValue_String_SP gStaticEidosValue_StringT;
 #define SLIM_TSK_INDIVIDUAL_REMEMBERED  ((uint32_t)(1 << 17))
 #define SLIM_TSK_INDIVIDUAL_FIRST_GEN   ((uint32_t)(1 << 18))
 
+// Metadata schemas:
+// These should be valid json strings, parseable by python's json.loads( )
+// and then turned into a valid metadata schema by tskit.MetadataSchema( ).
+// You can check these by doing, in python:
+// ```
+// t = ( <paste in everything below except final semicolon> )
+// d = json.loads(t)
+// m = tskit.MetadataSchema(d)
+// for e in d['examples']:
+//    m.encode_row(e)
+// ```
+// Furthermore, so that they match with the way python would do it,
+// we've produced these by doing :
+// ```
+// import pyslim
+// for ms in pyslim.slim_metadata_schemas:
+//   print(ms)
+//   print(str(pyslim.slim_metadata_schemas[ms]))
+// ```
+// See the pyslim code for readable versions of these.
+ 
+
+const std::string SLIM_TSK_METADATA_SCHEMA =
```

Comment:
Ah, I didn't notice this before: these strings should all be defined in `slim_globals.cpp`, not `.h`.  In this `.h` you just want a declaration, not a definition.  Search for `gStr_initializeAncestralNucleotides` to see an example of this, in the same source files.

Reply:
Also, maybe it would be better for these things to be in `slim_sim.h|cpp`?  That is the only place they're used, and the SLiMSim class is the responsible for this kind of thing – reading and writing `.trees` files.  So it probably isn't global enough to belong in `slim_globals.h`.

Reply:
so, should I call them like `gStr_tskIndividualMetadataSchema` instead of `SLIM_TSK_INDIVIDUAL_METADATA_SCHEMA`?

Reply:
wait, and do I really? I don't then do I use the gStr or the gID version where I want to use them, e.g. [here](https://github.com/MesserLab/SLiM/pull/101/files#diff-164c8a503989aafdda54a6ad5ffc8b9fR6256)?

Reply:
No, no, sorry.  I was just pointing you to the way those other const std::string objects are declared in the header and defined in the .cpp file.  The `extern` keyword and such.  No, the names of the variables are fine (if a bit non-standard; usually all caps is reserved for preprocessor defines, in C/C++).

Reply:
we could rename them... suggestions?

Reply:
Maybe this one would be gSLiM_tsk_metadata_schema?  Or gSLiM_TSK_metadata_schema?